### ISSUE TITLE
QR Code to open channel not working, no error message given

### ISSUE THREAD

**Issue by Engelberg** [btcontract/lnwallet#119]
I'm following the workflow to add an *inbound* channel using Bitrefill's new Thor service.

They display a QR Code for opening a 300,000 satoshi lightning channel supplied from their end.  In the wallet, I click open new channel and it shows me a list of peers to select.  Instead, I click to scan qr code and scan their qr code.  The wallet returns me to the peer selection screen with no other feedback about what is going wrong.

I am confused by the lack of any error message. My on-chain funds is $0. Does that have something to do with the problem?  If so, what kind of on-chain funds do I need to accept an inbound channel?

**Comment by d4amenace** [comment]
Is your node a brand new node? If so then you have a problem of unlocking the node. You must do some type of Channel opening first to feel the satoshi's security for each new channel.

**Comment by Engelberg** [comment]
No, it is not a brand new node.  I already have one open channel.

**Comment by btcontract** [comment]
Yes, this is sadly a known bug. In order for this to work QR code should be scanned from main page only (where the payment list is, by swiping screen to the left), or by opening a link on a phone.

Bitrefill currently instructs to go to "Open new channel" page first which is exactly the way to get this bug manifested and channel not open. I've fixed this but Google Play update will take a few days.

**Comment by Engelberg** [comment]
When I do it that way, a message pops up at the bottom saying "Requesting a channel...", and then the message fades a way after a few seconds and nothing happens -- no new channel, no further explanations.

**Comment by juscamarena** [comment]
can you try again?

**Comment by Engelberg** [comment]
Tried again (multiple times). Same outcome.

Am I expected to have on-chain funds available to open an incoming channel?

**Comment by juscamarena** [comment]
Ah was this the giveaway link?
If so it might have already been claimed?

**Comment by juscamarena** [comment]
If not can you please email the link to the channel here: justin@bitrefill.com

**Comment by Engelberg** [comment]
Yes, this was the giveaway link.  I re-clicked on the giveaway link before the last test, to ensure I wasn't using stale information.  Nothing to indicate the link was no longer valid.

**Comment by juscamarena** [comment]
A page refresh will show it's already been taken, but we should fix to have it check every x time. Sorry about that!

**Comment by Engelberg** [comment]
Refreshing does not show it has been taken. I've emailed you the link.
